src/App.tsx

```diff
@@ -15,47 +16,115 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+type ReorderOptions = {
+  sortType: SortType,
+  isReversed: boolean,
+};
+
+export function getReorderedGoods(
+  goods: string[],
+  { sortType, isReversed }: ReorderOptions,
+) {
+  const visibleGoods = [...goods];
+
+  switch (sortType) {
+    case SortType.ALPHABET:
+      visibleGoods.sort((a, b) => a.localeCompare(b));
+      break;
+    case SortType.LENGTH:
+      visibleGoods.sort((a, b) => a.length - b.length);
+      break;
+    default: visibleGoods; //eslint-disable-line
```

Comment:
Why you disable this line?

Reply:
I have an eslint error: Expected an assignment or function call and instead saw an expression.eslint[@typescript-eslint/no-unused-expressions]

Reply:
so, just put a break to the default case